.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,11 @@
+name: Validate Code Formatting
+on: [push]
+jobs:
+  validate-code-formatting:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - run: yarn
+      # This runs the `prettier` binary inside `node_modules/.bin`.
+      - run: yarn prettier --check src
```

Comment:
what kind of logs happened here?

Reply:
It will do something like this:

```shell
$ yarn prettier --check src
yarn run v1.22.11
$ /home/imballinst/codes/password-ga/node_modules/.bin/prettier --check src
Checking formatting...
[warn] src/App.js
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command
```

So, two things will be outputted: (1) the file names, and (2) the error code. I _think_ the error code, if non-zero, will mark the GHA as failed, but I might be wrong.

Reply:
...probably it's a good idea to add it as a comment. I'll do it.

Reply:
Updated! PTAL on the comments above and let me know if they are overly detailed or not.

Reply:
nice done